src/utils/helpers/getRandomIntInclusive.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ *
+ * Gets a random integer between min (inclusive) and max (inclusive).
+ *
+ * Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+ *
+ * @param min The minimum value in the range.
+ * @param max The maximum value in the range.
+ */
+export function getRandomIntInclusive(min: number, max: number): number {
+  if (min > max) {
+    throw new Error(`Min is greater than max. Min: ${min}. Max: ${max}.`);
+  }
+  const minCeiled = Math.ceil(min);
```

Comment:
If we're adding hard error-checking, I rather feel like passing non-integers should also throw an error, checked by `Number.isInteger`. Alternatively, you could use the integer-type tag that exists and force all callsites to at least assert that they're passing in an integer.

Reply:
Once you know they're integers, you don't need to floor and ceil.